### ISSUE TITLE
Import Error

### ISSUE THREAD

**Issue by Debianov** [psycopg/psycopg3#59]
Good day!

When importing the psycopg3 the following error occurs:
```
  File "d:\****\main.py", line 2, in <module>
    import psycopg3
  File "D:\Python\lib\site-packages\psycopg3\__init__.py", line 9, in <module>
    from . import pq
  File "D:\Python\lib\site-packages\psycopg3\pq\__init__.py", line 103, in <module>
    import_from_libpq()
  File "D:\Python\lib\site-packages\psycopg3\pq\__init__.py", line 95, in import_from_libpq
    raise ImportError(
ImportError: no pq wrapper available.
Attempts made:
- error importing 'c' wrapper: No module named 'psycopg3_c'
- error importing 'binary' wrapper: No module named 'psycopg3_binary'
- error importing 'python' wrapper: libpq library not found
```
The problem occurs on Windows 10 with PostreSQL 13 installed. According to the plan, libpq should be present...

**Comment by dvarrazzo** [comment]
Hello, thank you for trying psycopg3 out.

Eventually psycopg3 should have a binary bundle but it hasn't been released yet. In your current install layout the "python" wrapper should be the one working. I see it isn't because it cannot find the libpq, which should be a file called `libpq.dll` distributed together with Postgres.

Could you locate that file? Could you try and copy it in a directory included in the `PATH`?

This is just to try things out, it is not the way installation is supposed to work, of course :)

This is where importing the library takes place and where the exception is raised:

https://github.com/psycopg/psycopg3/blob/956238bef739675d46d6e65c953c1dcef10a165e/psycopg3/psycopg3/pq/_pq_ctypes.py#L16-L21

maybe there is something broken there you could check out?

Thank you!

**Comment by Debianov** [comment]
Hello!

I managed to solve the problem. First, as You requested, I checked PATH. There were no paths leading to libpq. After adding dll to one of the directories located in the PATH variable the process of importing the library did not go beyond 23 line in _pq_ctypes:

```python
pq = ctypes.cdll.LoadLibrary(libname)
```
The error said about dependencies:
```
Could not find module 'D:\Microsoft VS Code\bin\libpq.dll' (or one of its dependencies). Try using the full path with constructor syntax.
```
In the end, I started checking dependencies. Some of the dependent libraries were in the PostreSQL libraries folder. As a result, I had to add the path to PostreSQL/Lib in PATH variable. 

In conclusion, I would like to say that it would be nice to add this moment to the library documentation.

Thank you!
